c/lang/security/double-free.yaml

```diff
@@ -35,7 +35,7 @@ rules:
     category: security
     technology:
     - c
-    confidence: MEDIUM
+    confidence: LOW
```

Comment:
What is the rationale to move these rules from MEDIUM to LOW confidence? FP rate?